fdbserver/LogRouter.actor.cpp

```diff
@@ -264,6 +274,7 @@ ACTOR Future<Void> pullAsyncData( LogRouterData *self ) {
 					if(r) tagPopped = std::max(tagPopped, r->popped());
 					if( self->logSystem->get() ) {
 						r = self->logSystem->get()->peekLogRouter( self->dbgid, tagAt, self->routerTag );
+						self->primaryPeekLocation = r->getPrimaryPeekLocation();
```

Comment:
Can we determine if this log is on satellite or primary tlogs and report it as well? Note that log router don't always peek on satellite tlogs.

Reply:
It's not straightforward to get the locality of the specific log server from here. And with this UID, one should be able to determine whether the log is in satellite or not from another search pretty quickly.